packages/react/src/useTableUtils/types.ts

```diff
@@ -35,6 +35,7 @@ export interface TableOptions {
   pageSize?: number;
   initialCursor?: string;
   initialDirection?: "forward" | "backward";
+  initialSort?: { [column: string]: SortOrder };
```

Comment:
There is already a `sort` prop in `useTable`. We should have either `sort` or `initialSort`.

Reply:
Updated to omit the `sort` param, although I find the resulting type less helpful than it could be.